### DISCUSSION TITLE
smoke

### DISCUSSION THREAD

**Discussion by zzlcherry** [firemodels/fds#13429] [Q&A]
I want to achieve the effect of smoke control in FDS,but i keep changing the smoke exhaust volume and still can't achieve this goal.I don't know what has gone wrong. Could you please give me some answers?

**Comment by mcgratta** [comment]
Can you please provide more information, like a simple example.

**Comment by drjfloyd** [comment]
In addition to Kevin's comment make sure you are looking at quantitative performance measures (soot density, visibility, temperature, etc.) tied the specifics of your geometry, demographics, and egress times and not just the Smoke3D visualization in Smokeview.

**Reply by zzlcherry** [reply]
OK THANK YOU !

**Comment by mcgratta** [comment]
This case does not even run with the latest version of FDS. Remove the `BAROCLINIC` parameter. Then install the latest version of FDS and read the User's Guide section regarding special considerations in tunnels.

**Reply by zzlcherry** [reply]
Many thanks for your help1
